internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -106,6 +106,11 @@
     If specified, the -collector_version and -stage parameters will be ignored.
     .EXAMPLE
     .\install.ps1 -access_token "ACCESSTOKEN" -msi_path "C:\SOME_FOLDER\splunk-otel-collector-1.2.3-amd64.msi"
+.PARAMETER msi_public_properties
+    (OPTIONAL) Specify public MSI properties to be used when installing the Splunk OpenTelemetry Collector MSI package.
+    For information about the public MSI properties see https://learn.microsoft.com/en-us/windows/win32/msi/property-reference#configuration-properties
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -msi_public_properties "ALLUSERS=1" 
```

Comment:
What is the syntax for multiple properties? Space-separated `key=value` pairs?

Reply:
Here is a good example with 2 properties from https://learn.microsoft.com/en-us/windows/win32/msi/command-line-options

```DOS
msiexec /i testdb.msi INSTALLLEVEL=3 /l* msi.log COMPANYNAME="Acme ""Widgets"" and ""Gizmos."""
```

Reply:
Just realized that the command-line takes even private properties, will rename accordingly...